machinelearning/horovod/README.md

```diff
@@ -0,0 +1,127 @@
+# Horovod
+
+Horovod is a distributed training framework for TensorFlow, Keras, and PyTorch. The goal of Horovod is to make distributed Deep Learning fast and easy to use.
+
+This is a port of the Dockerfile maintained at https://github.com/uber/horovod
```

Comment:
Need to add the hyperlink here -> [https://github.com/uber/horovod](https://github.com/uber/horovod)